utils/docker/Dockerfile.centos.8

```diff
@@ -162,6 +162,7 @@ RUN dnf -y upgrade && \
         python3-Cython \
         python3-devel \
         python3-distro \
+        python3-pyelftools \
```

Comment:
this will break workflow builds, it needs to be installed directly with pip (as it currently is on master)